### PR TITLE
TOOL-140: fix SSO user roles

### PR DESCRIPTION
**Pull request by ketewan**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Fix bug allowing access to any db with sso

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nocodb/src/lib/meta/api/userApi/initStrategies.ts [hunk 1]
```diff
@@ -302,7 +302,7 @@ export function initStrategies(router): void {
                 } else {
                   let roles = 'editor';
 
-                  if (!(await User.isFirst())) {
+                  if (await User.isFirst()) {
```
Comment (Faradey27) [inline packages/nocodb/src/lib/meta/api/userApi/initStrategies.ts:305]: Why PR is draft?

did we had such a bug? totally opposite condition?

Comment (ketewan) [reply packages/nocodb/src/lib/meta/api/userApi/initStrategies.ts]: I rushed the fix, that's why turned into draft 
Now should be more realistic solution

**Review by Faradey27 — APPROVED** [review]
Code looks good.
Not sure how it fixes the problem

**Review by delenamalan — APPROVED** [review]
Agreed with @Faradey27 . Not sure how this fixes the issue, but worth a shot.

**Pull request merged**